miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt

```diff
@@ -65,11 +64,10 @@ abstract class UserInfoBridgeDispatcher {
 
     internal fun onGetUserName(callbackId: String) {
         try {
-            var isPermissionGranted = false
-            customPermissionCache.readPermissions(miniAppId).pairValues.find {
-                it.first == MiniAppCustomPermissionType.USER_NAME &&
-                        it.second == MiniAppCustomPermissionResult.ALLOWED
-            }?.let { isPermissionGranted = true }
+            val isPermissionGranted = customPermissionCache.hasPermission(
+                miniAppId,
+                MiniAppCustomPermissionType.USER_NAME
+            )
 
             if (isPermissionGranted) {
```

Comment:
you can call `customPermissionCache.hasPermission` here, just like what you did in `MiniAppChromeClient`.